app/assets/stylesheets/_footer.scss

```diff
@@ -1,38 +1,23 @@
 .footer__container {
-  position: relative;
+  position: absolute;
   bottom: 0;
   left: 0;
   height: 45px;
   width: 100%;
+  margin: 0 auto;
+  background-color: $grey-light;
```

Comment:
This shouldn't be necessary. We have this background-color set on `.main__wrapper`, so we don't need to repeat it here.